### ISSUE TITLE
compiler error looking for Binaries to get around

### ISSUE THREAD

**Issue by DevinBayly** [MRtrix3/mrtrix3#330]
Hi there,

I'm trying to install mrtrix on a cluster my school has set aside for research and the technical staff involved in installing software have not succeeded in compiling MRtrix because their systems are very out of date.

Is there a way that I can get access to a binary release for REHL 6.0 or 6.1? That's all I need. Thank you for your time!

**Comment by jdtournier** [comment]
At the moment, we don't supply binary builds - although we probably will once we get to the stage of creating a proper full release (i.e. version 3.0), hopefully in the next few months. In the meantime, the simplest thing to do is to set up a small virtual machine with an up to date OS, and compile static binaries as explained in the [standalone installation wiki page](https://github.com/MRtrix3/mrtrix3/wiki/linux-standalone-installation).

**Comment by draffelt** [comment]
I wonder if we could use the digital ocean web server for this? Rami, what
version of linux does it run?

**Comment by jdtournier** [comment]
@draffelt : are we talking about providing nightly builds...? Or just for the releases proper?

**Comment by draffelt** [comment]
I always thought binaries for release would be appropriate. However, if we
wanted to provide up to date binaries, we could probably get TravisCI to
have 1 of the test environments build static and push them to Github if all
tests pass?

But it would be for Mac and Linux only. Does the static build work on Mac?

**Comment by draffelt** [comment]
TravisCI runs 12.04

**Comment by draffelt** [comment]
64bit

**Comment by jdtournier** [comment]
I'm surprised TravisCI runs an outdated distro... I'm guessing you've had to install all the dependencies from non-standard sources? Even the compiler needs to be updated from a PPA to build MRtrix3 on 12.04...

In any case, I'm not sure I'd be up for providing binaries produced on an older distro - it would be OK if it was 14.04, although issues with Qt have been reported on that distro... I'd rather it was Arch Linux, but I appreciate that wouldn't be everyone's first choice... In any case, I reckon the DigitalOcean server would be better for this - assuming it runs something more recent, of course.

**Comment by rtabbara** [comment]
DigitalOcean server is running Ubuntu 14.04.1 LTS

**Comment by jdtournier** [comment]
Sounds like a better option. Only snag will be setting up a static build of Qt to link against - I doubt Canonical provide a ready-made static version of it...

**Comment by draffelt** [comment]
Yep, it's a tad annoying. I have to install the dependencies, including
gcc, from a PPA every test...
